src/ui/utils/apolloClient.tsx

```diff
@@ -53,9 +53,9 @@ export function ApolloWrapper({ children }: { children: ReactNode }) {
     );
   }
 
-  if (!isTest() && loading) {
-    return <StaticLoadingScreen />;
-  }
+  // if (!isTest() && loading) {
+  //   return <StaticLoadingScreen />;
+  // }
```

Comment:
?

Reply:
We have several places that hijack the flow to show loading screens that prevent the recording page SSR page from running. I'm commenting them out for now while this PR is in draft to get something running.